documentation/sphinx/source/command-line-interface.rst

```diff
@@ -114,6 +114,20 @@ perpetual storage wiggle
 
 Set the value speed (a.k.a., the number of processes that the Data Distributor should wiggle at a time). Currently, only 0 and 1 are supported. The value 0 means to disable the perpetual storage wiggle. For more details, see :ref:`perpetual-storage-wiggle`.
 
+storage migration type
+^^^^^^^^^^^^^^^^^^^^^^
+
+Set the storage migration type, or how FDB should migrate to a new storage engine if the value is changed.
+The default is ``disabled``.
+
+* ``disabled``
+* ``gradual``
+* ``aggressive``
+
+``gradual`` replaces a single storage at a time when the ``perpetual storage wiggle`` is active. This requires the perpetual storage wiggle to be set to a non-zero value to actually migrate storage servers. It is somewhat slow but very safe.
+``aggressive`` tries to replace as many storages as it can at once, and will recruit a new storage server on the same process as the old one. The main benefit over ``gradual`` is that this doesn't need to take one storage out of rotation, so it works for small or development clusters that have the same number of storage processes as the replication factor.
```

Comment:
if a process has two SSes, will that cause OOM to the process in a loaded cluster?

Reply:
Can you pls document the cons for aggressive as well?

Reply:
This is basically just assigning names to the existing mechanisms, and adding better control over them.
"aggressive" is the "wrongStoreTypeRemover" in DD, and "gradual" is "use the perpetual wiggle".
I can expand the documentation to elaborate on this more though.